packages/core/src/layout/Header/Waves.js

```diff
@@ -94,8 +93,8 @@ const Waves = ({ theme }) => {
           y2="241.925"
           gradientUnits="userSpaceOnUse"
         >
-          <stop stopColor={color2} stopOpacity="0.72" />
-          <stop offset="1" stopColor={color2} stopOpacity="0" />
+          <stop stopColor={color2} stopOpacity="35%" />
+          <stop offset="1" stopColor={color2} stopOpacity="20%" />
```

Comment:
Hmm, so this'll change the existing gradients, which probably isn't gonna look good. Could you look at recalculating the colors so that they fit the existing opacity? Or event recalculate all colors to get rid of the opacity altogether :grin:

Reply:
Sorry, my mistake. Yeah, this will change the existing gradients for all the colors. I've updated the screenshot with the reverted changes.